src/overlays/actors/ovl_En_Dt/z_en_dt.c

```diff
@@ -28,59 +98,725 @@ ActorInit En_Dt_InitVars = {
+
+    if ((gSaveContext.save.day == 3) && (gSaveContext.save.isNight)) {
+        EnDt_SetupFinalNightState(this, play);
+    } else {
+        s32 csId = this->actor.csId;
+        s32 i = 0;
+
+        // clang-format off
+        while (csId != CS_ID_NONE) { this->csIds[i] = csId; csId = CutsceneManager_GetAdditionalCsId(csId); i++; }
+        // clang-format on
+    }
+    this->actionFunc = EnDt_SetupCutsceneNpcs;
+}
+
+void EnDt_Destroy(Actor* thisx, PlayState* play) {
+    EnDt* this = THIS;
+
+    Collider_DestroyCylinder(play, &this->collider);
+}
+
+void EnDt_UpdateHeadRotate(EnDt* this) {
+    s32 yaw = ABS_ALT(BINANG_SUB(this->actor.yawTowardsPlayer, this->actor.world.rot.y));
+
+    this->headRotTarget.y = 0;
+    if (yaw < 0x4E20) {
+        this->headRotTarget.y = BINANG_SUB(this->actor.yawTowardsPlayer, this->actor.world.rot.y);
+
+        if (this->headRotTarget.y > 0x2710) {
+            this->headRotTarget.y = 0x2710;
+        } else if (this->headRotTarget.y < -0x2710) {
+            this->headRotTarget.y = -0x2710;
+        }
+    }
+}
+
+void EnDt_ChangeAnim(EnDt* this, s32 animIndex) {
+    f32 morphFrames;
+
+    this->animIndex = animIndex;
+
+    morphFrames = -4.0f;
+    if ((this->animIndex == EN_DT_ANIMATION_2) || (this->animIndex == EN_DT_ANIMATION_5)) {
+        morphFrames = 0.0f;
+    }
+
+    this->animEndFrame = Animation_GetLastFrame(sAnimations[this->animIndex]);
+    Animation_Change(&this->skelAnime, sAnimations[this->animIndex], 1.0f, 0.0f, this->animEndFrame,
+                     sAnimationModes[this->animIndex], morphFrames);
+}
+
+void EnDt_UpdateVisualState(EnDt* this) {
+    s32 index = this->visualState * 4;
+
+    index++;
+    EnDt_ChangeAnim(this, sVisualStateTable[index]);
+
+    index++;
+    this->eyeTexIndex = sVisualStateTable[index];
+
+    index++;
+    this->disableBlinking = sVisualStateTable[index];
+}
+
+void EnDt_UpdateCutsceneFocusTarget(EnDt* this) {
+    if ((this->npcEnMuto != NULL) && (this->npcEnBaisen != NULL)) {
+        EnMuto* npcEnMuto = (EnMuto*)this->npcEnMuto;
+        EnBaisen* npcEnBaisen = (EnBaisen*)this->npcEnBaisen;
+
+        switch (sCutsceneFocusTargetTable[this->textIdIndex]) {
+            case EN_DT_CS_FOCUS_TARGET_MUTO:
+                npcEnMuto->targetActor = this->npcEnMuto;
+                npcEnBaisen->targetActor = this->npcEnMuto;
+                this->targetActor = this->npcEnMuto;
+                break;
+            case EN_DT_CS_FOCUS_TARGET_BAISEN:
+                npcEnMuto->targetActor = this->npcEnBaisen;
+                npcEnBaisen->targetActor = this->npcEnBaisen;
+                this->targetActor = this->npcEnBaisen;
+                break;
+            case EN_DT_CS_FOCUS_TARGET_MAYOR:
+                npcEnMuto->targetActor = &this->actor;
+                npcEnBaisen->targetActor = &this->actor;
+                this->targetActor = &this->actor;
+                break;
```

Comment:
```suggestion
            case EN_DT_CS_FOCUS_TARGET_MUTO:
                npcEnMuto->targetActor = this->npcEnMuto;
                npcEnBaisen->targetActor = this->npcEnMuto;
                this->targetActor = this->npcEnMuto;
                break;

            case EN_DT_CS_FOCUS_TARGET_BAISEN:
                npcEnMuto->targetActor = this->npcEnBaisen;
                npcEnBaisen->targetActor = this->npcEnBaisen;
                this->targetActor = this->npcEnBaisen;
                break;

            case EN_DT_CS_FOCUS_TARGET_MAYOR:
                npcEnMuto->targetActor = &this->actor;
                npcEnBaisen->targetActor = &this->actor;
                this->targetActor = &this->actor;
                break;

            default:
                break;
```
Newlines please after breaks, makes it easier to tell different switch cases apart. If possible we like to add defaults as well if possible.